### PR TITLE
Gravity fix

### LINKED ISSUE #12: Get gravity working
Clean up any global variables in there while you're at it.

### PR DESCRIPTION
**Pull request by SuperShadowPlay**
Closes #12 

Gravity works again. All entities that interact gravitationally should be given a `gravitational_mass` value.
The baseline for this is that the player has a gravity of 1.
I opened all of the files and fixed most of the obvious global variables as well.

### CODE REVIEW

File: systems/RegisterBody.lua [hunk 1]
```diff
@@ -7,26 +7,32 @@ local Love = love
...
+  local new_body = Love.physics.newBody(
+    World,
+    pos_x + (body.offset_x or 0),
+    pos_y + (body.offset_y or 0),
+    body.type or 'dynamic'
+  )
 
-    return new_body
+  -- Set the body's mass
+  if mass then
```
Comment (jaythomas) [inline systems/RegisterBody.lua]: Where's this coming from?

**Review by jaythomas — CHANGES_REQUESTED** [review]
Correct me if I'm wrong but I don't think the ship_scaled.png asset is being used anywhere.

File: systems/RegisterBody.lua (hunk 1 above)
Comment (SuperShadowPlay) [reply systems/RegisterBody.lua]: I must have made the wrong call during the rebase of my branch. As as wise flash-cartoon character that wears boxing gloves once said:
_**DELETED**_

Comment (jaythomas) [reply systems/RegisterBody.lua]: As Coach Z would say... DELORTED

**Review by jaythomas — APPROVED** [review]

**Pull request merged**